tests/testthat/test-run_grid.R

```diff
@@ -193,7 +193,13 @@ test_that("compare baseline works as expected", {
   expect_equal(z$inc_primary[length(tt) - 1, 1],
                z$inc_vaccinated[length(tt) - 1, 1])
 
-  # number booster vaccinations is equal to
+  # number undergoing primary vaccination over 3 years is equal to the
+  # cumulative number for the 3 years
+
+  expect_equal(sum(z$inc_primary[, ]),
```

Comment:
```suggestion
  expect_equal(sum(z$inc_primary),
```
no need to use [,] notation when selecting whole array